app/Services/Plugins/ImporterService.php

```diff
@@ -383,14 +383,14 @@ protected function saveQuestElement(array $data, string $uuid, int $questId, Plu
     protected function importImage(MiscModel $model, PluginVersionEntity $entity)
     {
         // Don't do anything if no image or replacing an image (too many false positives)
-        if (empty($entity->image_path) || !empty($model->image)) {
+        if (empty($entity->image_path)) {
```

Comment:
You are now replacing every image when updating an entity by the one in the marketplace, which is exactly what we are trying to avoid :)